Build error on Linux: ‘EOVERFLOW_’ undeclared

steps to reproduce:
```
cd examples
make
```

output:
```
$ make
mkdir -p bin
gcc echo.c ../argtable3.c -I .. -o bin/echo
../argtable3.c: In function ‘arg_int_scanfn’:
../argtable3.c:2256:25: error: ‘EOVERFLOW_’ undeclared (first use in this function)
             errorcode = EOVERFLOW_;
                         ^
../argtable3.c:2256:25: note: each undeclared identifier is reported only once for each function it appears in
Makefile:61: recipe for target 'echo' failed
make: *** [echo] Error 1
```

--- a/argtable3.c
+++ b/argtable3.c
@@ -2253,7 +2253,11 @@ static int arg_int_scanfn(struct arg_int *parent, const char *argval)
         /* Safety check for integer overflow. WARNING: this check    */
         /* achieves nothing on machines where size(int)==size(long). */
         if ( val > INT_MAX || val < INT_MIN )
+#ifdef __STDC_WANT_SECURE_LIB__
             errorcode = EOVERFLOW_;
+#else
+            errorcode = EOVERFLOW;
+#endif
 
         /* Detect any suffixes (KB,MB,GB) and multiply argument value appropriately. */
         /* We need to be mindful of integer overflows when using such big numbers.   */

